strapi/src/guben-cockpit-plugins/cron-jobs/importer/main.ts

```diff
@@ -64,6 +63,25 @@ export class EventImporter {
     return categoriesIds;
   }
 
+  async getOrInsertLocationId (event: Event): Promise<string> {
+    const locationData = {
+      Street: event.location.street,
+      City: event.location.city,
+      Name: (event.location.name as any).DE ?? event.location.name,
```

Comment:
shouldn't this be as translatedString
that is the type that contains those DE, EN and PL properties